test/dump_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:test/test.dart';
+
+import 'utils/client.dart';
+
+void main() {
+  group('Dump', () {
+    setUpClient();
+
+    test('creates a dump', () async {
+      final dump = await client.createDump();
+
+      expect(dump.type, equals('dumpCreation'));
+      expect(dump.status, anyOf('succeeded', 'enqueued'));
```

Comment:
Would we not add a waitForTask to check that it ends up `succeeded`? Just a suggestion

Reply:
Yeah, I could add that, but in this case, I am just interested to know if the createDump() successfully enqueued a task.